translations/Arabic/special-terms.md

```diff
@@ -0,0 +1,48 @@
+# Terms
+
+- Message: رسالة
+  لقد وصلتك رسالة
+- Private Message (PM): رسالة خاصة
+  أرسل رسالة خاصة
+- Group PM: رسالة خاصة لمجموعة
+- Realm: نطاق
+  Closest translation possible
+- Stream: بث
+  Direct translation would be something like river stream. بث (which means something like TV broadcast) is the best translation in the context
+  شاهد البث المباشر
+- Topic: موضوع
+- Private/Invite-only Stream: بث خاص\بدعوة
+  يجب أن يكون لديك دعوة لتنضم لهذا البث الخاص
+- Public Stream: بث عام
+- Bot: روبوت
+  There is no translation for this word, so the Arabic writing for 'Robot' is used
+- Integration: تكامل
+- Notification: إشعار
+  وصلك إشعار جديد
+- Alert Word: كلمة تنبيهية
+- View: شاهد
```

Comment:
I'm not sure it's the right term in the context of web apps, wouldn't `عرض` be more appropriate here? 

I think doing task type A would've been beneficial here, since with so many synonyms it's hard to tell which words are commonly used in web apps.